tunder/test/helpers.dart

```diff
@@ -80,3 +84,16 @@ Future<void> assertDatabaseDoesntHave(
       reason:
           "Database table [$table] shouldn't have records matching $where but found $count record(s)");
 }
+
+parseBody(body, Headers? headers) {
+  if (isJson(headers)) return jsonEncode(body);
+
+  return body;
+}
+
+bool isJson(Map<String, String>? headers) {
+  if (headers == null) return false;
+
+  return headers['content-type'] == 'application/json' ||
+      headers['Content-Type'] == 'application/json';
```

Comment:
Reading HTTP/1.1's RFC, it seems like "Field names are case-insensitive.", but this would be a framework-wide change I guess. 

https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2